applications/espresso.yml

```diff
@@ -0,0 +1,93 @@
+name: espresso
```

Comment:
Let's add a YAML linter for the PRs pls, @chu-k already added for https://github.com/Exabyte-io/python_code/blob/main/.github/workflows/cicd.yml